osdep/io.h

```diff
@@ -172,6 +172,9 @@ void mp_globfree(mp_glob_t *pglob);
 #undef fstat
 #define fstat(...) mp_fstat(__VA_ARGS__)
 
+#define utime(...) _utime(__VA_ARGS__)
+#define utimbuf _utimbuf
+
```

Comment:
Why does this need a _ prefix while similar functions seem to be available without?

Reply:
I don't know much about the API exposed on windows, but [this question](https://stackoverflow.com/q/37845163/945780) supposes it's about ISO compliance.

Reply:
The different names are just bullshit MS made up. Most of them are available unprefixed, somehow, I think.